cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,74 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
+import ArticleEditorPageObject from '../support/pages/articleEditor.pageObject';
+import ArticlePageObject from '../support/pages/articlePage.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+
+const articleEditorPage = new ArticleEditorPageObject();
+const articlePage = new ArticlePageObject();
+const homePage = new HomePageObject();
+
 describe('Article', () => {
-  before(() => {
+  let user;
+  let article;
+
+  const articleChaned = {
+    title: faker.lorem.words(),
+    description: faker.lorem.words(),
+    body: faker.internet.email()
```

Comment:
The 'body' of the article is being generated with 'faker.internet.email()', it should be 'faker.lorem.paragraph()' or similar function which generates text, not an email.